build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     }
 
     dependencies {
-        classpath deps.gradle_plugins.android
+        classpath 'com.android.tools.build:gradle:4.1.1'
```

Comment:
There's no need to remove variable for the build gradle tools dependency with a hardcoded dependency.